package.json

```diff
@@ -12,17 +12,15 @@
     "console": "sst console",
     "lint": "tsc && eslint '**/*.js'",
     "clean": "rm -rf dist node_modules package-lock.json ./*/{.cache,dist,node_modules}",
-    "test": "npm test -w billing -w upload-api -w carpark -w replicator -w satnav -w ucan-invocation -w roundabout -w filecoin",
```

Comment:
we do not have that package anymore since https://github.com/web3-storage/w3infra/pull/315